File: JavaScript/프로그래머스/1/86491. 최소직사각형/최소직사각형.js
function solution(sizes) {
    for (let size of sizes) {
        size.sort((a, b) => a - b);
    }
    
    let max_w = 0;
    let max_h = 0;
    for (let i=0; i<sizes.length; i++) {
        if (sizes[i][0] > max_w) max_w = sizes[i][0];
        if (sizes[i][1] > max_h) max_h = sizes[i][1];
    }
    
    return max_w * max_h;
}

File: Python3/프로그래머스/1/86491. 최소직사각형/최소직사각형.py
def solution(sizes):
    for size in sizes:
        size.sort()
    
    max_w, max_h = sizes[0]
    for i, j in sizes:
        if i > max_w:
            max_w = i
        if j > max_h:
            max_h = j
            
    return max_w * max_h